FrontEnd/src/components/Contact/Contact.jsx

```diff
@@ -1,28 +1,201 @@
-import React from 'react';
-import renderContent from '../../pages/CookiesPolicyPage/RenderContent.jsx';
+import { useState, useEffect, useRef } from 'react';
+import { useForm, Controller } from 'react-hook-form';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import { Spin, Select, Space } from 'antd';
+
+import {
+    EMAIL_PATTERN,
+    MESSAGE_PATTERN
+} from '../../constants/constants';
+
 import LinkContainer from '../../pages/CookiesPolicyPage/LinkContainer.jsx';
+
 import styles from './Contact.module.css';
-import contactText from './text';
-import TEXT_CONTENT from './text';
-import useScrollToTop from '../../hooks/useScrollToTop';
+
+const CATEGORY_OPTIONS = [
+    { value: 'Технічне питання', label: 'Технічне питання' },
+    { value: 'Рекомендації', label: 'Рекомендації' },
+    { value: 'Питання', label: 'Питання' },
+    { value: 'Інше', label: 'Інше' },
+];
 
 const Contact = () => {
-  useScrollToTop();
-
-  return (
-    <div className={styles['contact_container']}>
-      <div className={styles['contact__link_container']}>
-        <LinkContainer />
-        <img className={styles['contact__img1']}
-          src={`${process.env.REACT_APP_PUBLIC_URL}/img/dots_12x10.png`}
-          alt="dots_12x10.png" />
-      </div>
-      <div className={styles['contact__text_container']}>
-        <h2 className={styles['contact__title']}>{contactText.title}</h2>
-        {renderContent(TEXT_CONTENT)}
-      </div>
-    </div>
-  );
+    const [loading, setLoading] = useState(false);
+    const [showModal, setShowModal] = useState(false);
+    const [modalMessage, setModalMessage] = useState('');
```

Comment:
nit: Do we need these two states? We can show the modal if `modalMessage` contains text and clean on close.

Reply:
fixed it, thank you